packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card-icons.scss

```diff
@@ -0,0 +1,98 @@
+@use '~genesys-spark/dist/scss/checkbox-icons.scss';
+@use '~genesys-spark/dist/scss/radio-icons.scss';
+
+@mixin radio-unchecked {
+  @include radio-icons.radio-unchecked;
+
+  background-color: var(
+    --gse-ui-selectorCard-default-selectedIndicator-unselected-foregroundColor
+  );
```

Comment:
Is `background-color` the only property that is causing this file to be needed ? Would it be better to delete this file and just put the `background-color` under the mixin in the `gux-selector-card.scss` file like ,

`::slotted(input[type='radio']:not(:checked))::before {
  background-color: var(
    --gse-ui-selectorCard-default-selectedIndicator-unselected-foregroundColor
  );
  @include gux-selector-card-icons.radio-unchecked;
}`